charts/dynamoai-iam/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for dynamoai.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  nameOverride: "dynamoai"
+  fullnameOverride: "dynamoai"
+  serviceAccount:
+    name: "dynamoai-service-account"
+  secrets:
+    mongodb: dynamoai-mongodb-secret
+    postgres: dynamoai-postgres-secret
+    redis: dynamoai-redis-secret
+    common: dynamoai-common-secret
+  config:
+    common: dynamoai-common-config
+    proxy: dynamoai-proxy-config
+  awsRegion: "us-east-1"
+
+keycloak:
+  enabled: true
+  fullnameOverride: keycloak
+  auth:
+    adminUser: "admin"
+    adminPassword: "" # TODO
+
+  adminIngress:
```

Comment:
I know I've not updated this in the pulumi too but in certain cases e.g prod SAAS we don't have ingress enabled. There we might have to go via a different route.

OR @suvom-das Can we always have ingress from now on?
We anyways plan to switch to a new prod deployment with helm, we can ensure we keep ingress enabled in that deployment.

cc: @brarkaran